### ISSUE TITLE
bioy blast unit test

### ISSUE THREAD

**Issue by tyleraland** [nhoffman/bioy#39]
Proposal:

Store a tiny blast database with the test files
Query sequences against this custom database

Unit tests:

* sequence A finds identical sequence B in the database
* sequence A finds tiny sequence B in the database with high identity, low coverage requirement
* sequence A does not find tiny sequence B in the database with high identity because its low coverage doesn't meet the minimum required

**Comment by nhoffman** [comment]
agreed that this would be useful - the stored blast output could be used for other tests as well
